src/Command/CreateClassCacheCommand.php

```diff
@@ -26,6 +26,8 @@
 /**
  * NEXT_MAJOR: Remove this class.
  *
+ * @deprecated since version 3.39.0 and will be removed in 4.0.
```

Comment:
I'll replace and adapt this on release. Also, let's add the package name.

```suggestion
 * @deprecated since version sonata-project/admin-bundle 3.x and will be removed in 4.0.
```

You're not the first one to do this, and I'm starting to wonder if there is something somewhere telling people to use fixed numbers

Reply:
I think this is a good practice which is used in most projects.

Reply:
Makes sense.

Reply:
Keep in mind that this class was deprecated on 3.39. Look at line 22

Reply:
Should we remove the trigger deprecation, or is it correct?

Reply:
It's correct. Trigger error throws an error at runtime but deprecated tag is just for IDE.

Reply:
So should I apply these changes or leave as is?

Reply:
If it's already released, you can leave as is.

Reply:
Wait actually, please apply the comment about the package name